applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.cpp

```diff
@@ -51,13 +51,25 @@ void GeoLinearElasticPlaneStrain2DLaw::GetLawFeatures(Features& rFeatures)
     rFeatures.mSpaceDimension = WorkingSpaceDimension();
 }
 
+SizeType GeoLinearElasticPlaneStrain2DLaw::WorkingSpaceDimension() {
+    return Dimension;
+}
+
+SizeType GeoLinearElasticPlaneStrain2DLaw::GetStrainSize() const {
+    return VoigtSize;
+}
+
+bool GeoLinearElasticPlaneStrain2DLaw::IsIncremental() {
+    return true;
+}
+
 void GeoLinearElasticPlaneStrain2DLaw::CalculateElasticMatrix(Matrix& C, ConstitutiveLaw::Parameters& rValues)
 {
     KRATOS_TRY
 
     const Properties &r_material_properties = rValues.GetMaterialProperties();
-    const double &  E = r_material_properties[YOUNG_MODULUS];
-    const double & NU = r_material_properties[POISSON_RATIO];
+    const double &  E  = r_material_properties[YOUNG_MODULUS];
+    const double &  NU = r_material_properties[POISSON_RATIO];
```

Comment:
Nitpicking: it is rather uncommon to use references to const objects for built-in types such as `double`. In these cases I would prefer values over references. Furthermore, I would use `auto` here and rename `NU` such that is follows the Kratos Style Guide:
```suggestion
    const auto E  = r_material_properties[YOUNG_MODULUS];
    const auto nu = r_material_properties[POISSON_RATIO];
```

Reply:
Done